src/test/java/uk/gov/hmcts/reform/em/orchestrator/automatedbundling/BundleFactoryTest.java

```diff
@@ -156,6 +162,7 @@ public void createWithDocumentSetFilters() throws IOException, DocumentSelectorE
         BundleConfiguration configuration = new BundleConfiguration(
             "Bundle title",
             "filename.pdf",
+                "PageRange",
```

Comment:
please enum this with a camel case (its the convention we've used in CCD config so far)